samples/smoketest/smoke_test_async.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+from key_vault_secrets_async import KeyVault_async
+from event_hubs_async import EventHub_async
+
+print("")
+print("==========================================")
+print("   AZURE TRACK 2 SDKs SMOKE TEST ASYNC")
+print("==========================================")
+
+
+async def main():
+    await KeyVault_async().Run()
+    await EventHub_async().Run()
+
+
+loop = asyncio.get_event_loop()
+
+task = loop.create_task(main())
+loop.run_until_complete(task)
```

Comment:
No need to wrap the coroutine in a task yourself, you can write `loop.run_until_complete(main())`.